### ISSUE TITLE
Demonstrate initialising SpatialKappa Simulation with external simulator interface 

### ISSUE THREAD

**Issue by rjwen2045458** [rjwen2045458/KappaNEURON#9]

**Comment by davidcsterratt** [comment]
1. Put code from https://colab.research.google.com/drive/1WiHegXXOPO015fH_1ww7j5SlHBq43fT8?usp=sharing#scrollTo=OjDsCsaWptHJ into a python file.
2. Get `ExternalSimluatorInterface.init` to create a SpatialKappa simulation. See `kappa_sim = gateway.kappa_sim(...` in `KappaNEURON/__init__.py`
3. Test whether the script still works

One problem is that we may have a global "gateway" variable. Look at how this is done in `KappaNEURON/__init__.py`.

**Comment by rjwen2045458** [comment]
@davidcsterratt 
In the original KappaNEURON __init__.py file, there exists an important input "**kwargs". There are too many variables relates to it. The task is rewriting __init__ function of ExternalSimulatorInterface using code in _create_kappa_sims function of KappaNEURON. Actually, the ExternalSimulatorInterface class can not work if it completely avoids to use "**kwargs" related variables.

**Comment by davidcsterratt** [comment]
@rjwen2045458 Yes, I think the ExternalSimulator interface can take kwargs, for example the Kappa file name. I think the main thing to do is make sure that you can create a SpatialKappa simulation within the ExternalSimulator init function.
